tests/integration/targets/kubevirt_vm/playbook.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: Create VM
+  connection: local
+  gather_facts: false
+  hosts: localhost
+  tasks:
+    - name: Create a VirtualMachine
+      kubevirt.core.kubevirt_vm:
+        state: present
+        name: testvm3
+        namespace: default
+        labels:
+          app: test
+        wait: true
+        wait_timeout: 600
+        spec:
+          domain:
+            cpu:
+              sockets: 1
+            memory:
+              guest: 1536Mi
```

Comment:
Use `1Gi`?

Reply:
Nope :). Since it accepts an arbitrary number, it gets a very arbitrary one